locales-pending/fix.ftl

```diff
@@ -27,6 +27,11 @@ fix-flow-data-broker-profiles-view-data-broker-profiles-content = Data brokers m
 fix-flow-data-broker-profiles-view-data-broker-profiles-button-remove-for-me = Remove them for me
 fix-flow-data-broker-profiles-view-data-broker-profiles-button-remove-manually = I’ll remove them manually
 fix-flow-data-broker-profiles-view-data-broker-profiles-view-info-on-sites = View your info on these sites
+fix-flow-data-broker-profiles-view-data-broker-profiles-more-dialog-trigger-label = More info
+fix-flow-data-broker-profiles-view-data-broker-profiles-more-dialog-title = About these profiles
+fix-flow-data-broker-profiles-view-data-broker-profiles-more-dialog-paragraph1 = Data brokers make up a $240 billion industry of gathering your personal info and selling it to anyone searching for you. They scrape your data from publicly available sources to create profiles that include your name(s), current and previous addresses, family member names, criminal history, and much more.
+fix-flow-data-broker-profiles-view-data-broker-profiles-more-dialog-paragraph2 = We found these profiles by searching 190 data broker sites for the name, location, and date of birth you provided. These profiles are currently live and available to anyone searching for you.
```

Comment:
also for this, we shouldn't hardcode the "190" and I think we have this data broker count hardcoded somewhere in an exported const so we can update it in one place.

```sh
git grep -n "ONEREP_DATA_BROKER_COUNT =" | cat

src/app/functions/server/onerep.ts:93:export const ONEREP_DATA_BROKER_COUNT = 190;
```